src/analysis/retail/evoker/preservation/modules/talents/Echo.tsx

```diff
@@ -67,6 +97,36 @@ class Echo extends Analyzer {
     return isFromTAEcho(event);
   }
 
+  get suggestionThresholds() {
+    return {
+      actual: this.percentWasted,
```

Comment:
I just did percentages because defining thresholds is a bit tricky based on fight length. I can change it tho and just do 1,3,5 if you think thats better.